ecosystem/typescript/sdk/src/token_client.ts

```diff
@@ -278,16 +278,40 @@ export class TokenClient {
    * ```
    */
   async getTokenBalance(creator: MaybeHexString, collectionName: string, tokenName: string): Promise<Types.Token> {
```

Comment:
I'd argue we shouldn't really have this function. It is unclear why we would ever want a specific function to query the balance for the creators account.

Reply:
Yeah I'd agree.  Why not just call the per account one then?

Reply:
Let's create a github issue for these major version breaking changes that we want to do.

Reply:
@davidiw we shouldn't have this for sure. This is an old API. The reason I did not kill this is that I don't want to introduce breaking changes. We are going to make it away in the next major release.